lib/utils/HttpHelper.dart

```diff
@@ -61,7 +61,7 @@ class HttpHelper {
 
     // add all file entries to the request
     for (MapEntry<String, Uri> entry in files.entries) {
-      List<int> fileBytes = await UriReader.instance.readAsBytes(entry.value);
+      List<int> fileBytes = await UriReader.instance.readFileAsBytes(entry.value);
```

Comment:
Please stick to `readAsBytes`, we need to be able to read http resources too.

Reply:
@monsieurtanuki In that case the relative path will not work on tests, this is why in the first play I have used File().absolut.uri. 

I will change this back to readAsByte, and on the test I will make it again File().absolut.uri.

Reply:
That's why I suggested to change to algorithm in `UriReader` in a previous comment, so that relative `File`s work as well.
I tried that successfully with the `flutter test test` command - they say that it does not work that well with `flutter test`.
In the worst case, please rollback to your initial code with `absolute.uri` and `readAsBytes`.

Reply:
@monsieurtanuki I made the rollback.

Reply:
Thank you @MohamedFBoussaid! Sorry about these code changes back and forth.
I still think we would be better off with a `UriReader` that works with relative files - I'll fix that in another PR today.